rest/src/main/java/discord4j/rest/util/Permission.java

```diff
@@ -132,14 +129,42 @@ public enum Permission {
     /** Allows management and editing of emojis and stickers. */
     MANAGE_EMOJIS_AND_STICKERS(0x40000000, true),
 
-    /** Allows members to use slash commands in text channels. */
+    /** Allows members to use slash commands in text channels.
+     *
+     *  @deprecated Deprecated in favor of {@link Permission#USE_APPLICATION_COMMANDS}
+     */
+    @Deprecated
     USE_SLASH_COMMANDS(0x80000000L, false),
 
+    /** Allows members to use application commands, including slash commands and context menu commands. */
+    USE_APPLICATION_COMMANDS(0x80000000, false),
```

Comment:
It produces Integer overflow (and have bugfix  https://github.com/Discord4J/Discord4J/commit/22b408150b583a98135051f255a74f16947f8a50). Add `L` to the end of number